### PR TITLE
Removes the Chapel, Reverts God.

### PR DESCRIPTION
**Pull request by LordFowl**
Removes the chapel and replaces it with a holodeck.
Replaces the old holodeck with a pool/gymnasium.
Properly sets up the holodeck, mapwise. All holorooms have been adjusted to be the appropriate size for the now slightly smaller holodeck (7x10).
The Chaplain now has an office above hydroponics, which has all of his necessary things - crematorium, morgue tray, tools of office, and even a slight semi-confessional for private council.
Adds a chapel to the holodeck, for when the chaplain wishes to conduct a service. Additional holodecks for specific lore-religions are  pending.
Adds a gymnasium holoroom.

Probably should be DNM until Lohikar actually investigates why holodecks are broken, namely the potential issue with copy_contents_to(). I guess you can merge it if you're okay with the chaplain being temporarily bereft of his chapel, however.

### CODE REVIEW

File: code/game/turfs/flooring/flooring_premade.dm [hunk 1]
```diff
@@ -220,13 +220,19 @@
...
 	icon_state = "seadeep"
 
+/turf/simulated/floor/beach/water/pool
+	icon_state = "pool"
+	watertype = "poolwater"
+
 /turf/simulated/floor/beach/water/Initialize()
 	. = ..()
-	add_overlay(image("icon"='icons/misc/beach.dmi',"icon_state"="water5","layer"=MOB_LAYER+0.1))
+	if(add_overlay(image("icon"='icons/misc/beach.dmi',"icon_state"="[watertype]","layer"=MOB_LAYER+0.1)))
```
Comment (Lohikar) [inline code/game/turfs/flooring/flooring_premade.dm]: `add_overlay` does not return.

Comment (LordFowl) [inline code/game/turfs/flooring/flooring_premade.dm]: And yet it just works.

**Comment by LordFowl** [comment]
TGDMM merge seems to shit itself again trying to handle this. Causes some sort of overflow. Everything past "Zz" gets marked "OVERFLOWX"

**Comment by LordFowl** [comment]
With Lohikar's fix confirmed and pending, this can be merged.

**Comment by LordFowl** [comment]
And now map merge doesn't seem to be working at all. Hnnggg.

**Comment by LordFowl** [comment]
Well, I've fixed mapmerge's dictionary error again by using the old mapmerge, but that seems to have only slightly reduced the diff.

**Pull request merged**